core/css.js

```diff
@@ -499,6 +499,7 @@ Blockly.Css.CONTENT = [
 
   '.blocklyDropdownMenu {',
     'padding: 0 !important;',
+    'max-height: 300px !important;',
```

Comment:
Add a comment here saying that this is the same value as the constant that you use in `field_dropdown.js`.

Reply:
Done!